chromadb/api/types.py

```diff
@@ -1,4 +1,4 @@
-from typing import Optional, Union, TypeVar, List, Dict, Any, Tuple, cast
+from typing import Optional, Set, Union, TypeVar, List, Dict, Any, Tuple, cast
```

Comment:
nit - maybe worth breaking the validators / types out a bit

Reply:
I will extract them in a future PR. Will open an issue.

Reply:
added as a task on https://github.com/chroma-core/chroma/issues/2292